src/views/standalone/system/StorageView.vue

```diff
@@ -131,13 +147,16 @@ onMounted(() => {
             <div class="text-sm">
               <p>
                 <strong>{{ t('standalone.storage.size') }}:</strong>
-                {{ currentStorageConfiguration.size ?? t('standalone.storage.unknown') }}
+                {{
+                  byteFormat1000(parseInt(currentStorageConfiguration.size ?? '0')) ??
```

Comment:
`byteFormat1024` is better for disk sizes

Reply:
Fixed.